libs/substrate-differ/src/differs.rs

```diff
@@ -23,6 +24,9 @@ pub struct DiffOptions {
 	pub include: DiffOptionInclude,
 }
 
-pub trait Differ {
-	fn diff(&self, options: DiffOptions);
+pub trait Differ<T>
+where
+	T: PartialEq,
+{
+	fn diff(&self, options: DiffOptions) -> Vec<(String, u32, DiffResult<'_, T>)>;
```

Comment:
Can you explain the return value `(String, u32, DiffResult<'_, T>)`, especially the String/u32 ?
Wouldn't it better to extract a type here ?

Reply:
Ok it is for name + index. We have a type alias for the Index.

Reply:
As you can see [here](https://github.com/chevdor/subwasm/blob/wk-220711-differ-updated/libs/substrate-differ/src/differs/reduced/call/prelude.rs#L5), I saw it coming :)

Reply:
We had the `Index` type defined twice. One as u8 and one as u32. I removed the dup and now we have only one u32.